tests/aead_aes_128_gcm_truncated_tag_96_tests.txt

```diff
@@ -0,0 +1,235 @@
+# The AES GCM test vectors come from the file gcmDecrypt128.rsp
+# downloaded from http://csrc.nist.gov/groups/STM/cavp/index.html
+
+# test_group_0
+
+KEY = 11754cd72aec309bf52f7687212e8957
```

Comment:
Hi, why did you switch from the Chromium test vectors to these NIST test vectors? I was expecting you would adjust the Chromium test vectors to be in the same form as this. I think we should stick with the Chromium test vectors for this. Is there something missing from the Chromium test vectors that make it hard/impossible to adapt them to this form?

Reply:
sure, I use the test vectors from Chromium, but they comment it [come from NIST](https://chromium.googlesource.com/chromium/src/+/master/net/quic/core/crypto/aes_128_gcm_12_decrypter_test.cc#19).